tests/lib/Form/Data/Role/RoleAssignmentCreateDataValidationTest.php

```diff
@@ -54,6 +54,11 @@ public static function getDataForTestFormSubmitValidation(): iterable
                     ['{{ value }}' => 'foo'],
                     'children[limitation_type]'
                 ),
+                new FormErrorDataTestWrapper(
+                    'The selected choice is invalid.',
+                    ['{{ value }}' => 'foo'],
+                    'children[limitation_type]'
+                ),
```

Comment:
That codifies duplicate validation instead of addressing it. If the form only needs one message, this change will hide a regression and could show users two identical errors. But maybe it is not the case at all. What about changing the message in the second form error?